server/chat-commands.js

```diff
@@ -656,9 +656,10 @@ const commands = {
 
 	status(target, room, user, connection, cmd) {
 		if (!this.canTalk()) return;
-		if (!toID(target)) return this.parse('/help status');
+		if (!target) return this.parse('/help status');
+		if (!/[a-zA-Z0-9]/.test(target)) return this.errorReply("Your status must contain at least one letter or number.");
```

Comment:
Why?

Reply:
Which part are you asking about?

Reply:
Why must statuses contain at least one letter or number?

Reply:
Like, that's not actually an intended restriction; it was a restriction you seemingly accidentally added in #5577.